functions/ecom.config.js

```diff
@@ -371,6 +371,83 @@ const app = {
       },
       hide: false
     },
+    pix: {
+      schema: {
+        type: 'object',
+        title: 'Pix',
+        description: 'Configurações adicionais para Pix',
+        additionalProperties: false,
+        properties: {
+          enable: {
+            type: 'boolean',
+            default: false,
+            title: 'Habilitar Pix via Mercado Pago'
+          },
+          key_pix: {
+            type: 'string',
+            maxLength: 255,
+            title: 'Chave Pix Mercado Pago'
+          },
+          label: {
+            type: 'string',
+            maxLength: 50,
+            title: 'Rótulo',
+            description: 'Nome da forma de pagamento exibido para os clientes',
+            default: 'Pix'
+          },
+          text: {
+            type: 'string',
+            maxLength: 1000,
+            title: 'Descrição',
+            description: 'Texto auxiliar sobre a forma de pagamento, pode conter tags HTML'
+          },
+          icon: {
+            type: 'string',
+            maxLength: 255,
+            format: 'uri',
+            title: 'Ícone',
+            description: 'Ícone customizado para a forma de pagamento, URL da imagem'
+          },
+          min_amount: {
+            type: 'number',
+            minimum: 0,
+            maximum: 999999999,
+            title: 'Pedido mínimo',
+            default: 0,
+            description: 'Montante mínimo para listar meio de pagamento via Pix'
+          },
+          apply_at_discount: {
+            type: 'string',
+            enum: [
+              'total',
+              'subtotal',
+              'freight'
+            ],
+            default: 'subtotal',
+            title: 'Aplicar desconto em',
+            description: 'Em qual valor o desconto deverá ser aplicado no checkout'
+          },
+          type_discount: {
+            type: 'string',
+            enum: [
+              'percentage',
+              'fixed'
+            ],
+            default: 'percentage',
+            title: 'Tipo de desconto',
+            description: 'Desconto com valor percentual ou fixo'
+          },
+          value_discount: {
+            type: 'number',
+            minimum: -99999999,
+            maximum: 99999999,
+            title: 'Valor do desconto',
+            description: 'Valor percentual ou fixo a ser descontado, dependendo to tipo configurado'
+          }
```

Comment:
Por que você adicionou esses campos aqui em vez de um toggle _Desconto no Pix_ no objeto `discount` já definido nas configurações? Meio que já era um padrão desse app https://github.com/ecomplus/app-mercadopago/blob/2124f474def2b0b5fb65bb5882d6757f06955371/functions/ecom.config.js#L222-L233

Reply:
A ideia aqui era para o pix ter um desconto diferente dos demais.

Reply:
Acho que não faz muito sentido o lojista ter um desconto X% em cartão e Y% em pix 🙃 
Em geral ele vai dar desconto em uma forma de pagamento só, em uma ou outra ocasião pode ser que dê desconto em mais formas de pagamento (pix & boleto & cartão à vista) mas deve ser o mesmo desconto, caso contrário fica até confuso demais né...

Reply:
Meu ponto aqui sobre o objeto de configuração é tentar deixar mais intuitivo pro lojista quando for configurar na verdade, "complexar" o `admin_settings` não influencia só em como o código vai ter que tratar a config depois mas também em como o lojista vai configurar o app né